File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/Auto_OnlyHighGoal.java
package org.firstinspires.ftc.teamcode.opmodes;

import com.qualcomm.robotcore.eventloop.opmode.Autonomous;

import org.firstinspires.ftc.teamcode.math.Pose2D;
import org.firstinspires.ftc.teamcode.superclasses.WobbleManipulator;

import static org.firstinspires.ftc.teamcode.opmodes.MovementMacros.Park;
import static org.firstinspires.ftc.teamcode.opmodes.MovementMacros.ShootHighGoal;
import static org.firstinspires.ftc.teamcode.opmodes.MovementMacros.ShootTargets;
import static org.firstinspires.ftc.teamcode.robot.WoENrobot.movement;
import static org.firstinspires.ftc.teamcode.robot.WoENrobot.odometry;
import static org.firstinspires.ftc.teamcode.robot.WoENrobot.wobbleManipulator;

@Autonomous
public class Auto_OnlyHighGoal extends AutoOpMode {

    @Override
    public void main() {
        movement.Pos(new Pose2D(odometry.getRobotCoordinates().x + 15 * sideSign, -50, Double.NaN), 1, 1, 10, 0.1);
        ShootHighGoal();
        // MoveWobble();
        //movement.Pos(new Pose2D(getStartPosition().x, 25, 0));
        Park();
        wobbleManipulator.setAngle(WobbleManipulator.Position.UP);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/Auto_OnlyHighGoal.kt
package org.firstinspires.ftc.teamcode.opmodes

import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import org.firstinspires.ftc.teamcode.math.Pose2D
import org.firstinspires.ftc.teamcode.opmodes.MovementMacros.Park
import org.firstinspires.ftc.teamcode.opmodes.MovementMacros.ShootHighGoal
import org.firstinspires.ftc.teamcode.robot.WoENrobot.movement
import org.firstinspires.ftc.teamcode.robot.WoENrobot.odometry
import org.firstinspires.ftc.teamcode.robot.WoENrobot.wobbleManipulator
import org.firstinspires.ftc.teamcode.superclasses.WobbleManipulator

@Autonomous
class Auto_OnlyHighGoal : AutoOpMode() {
    override fun main() {
        movement.pos(Pose2D(odometry.robotCoordinates.x + 15 * sideSign, -50.0, Double.NaN), 1.0, 1.0, 10.0, 0.1)
        ShootHighGoal()
        // MoveWobble();
        //movement.Pos(new Pose2D(getStartPosition().x, 25, 0));
        Park()
        wobbleManipulator.setAngle(WobbleManipulator.Position.UP)
    }
}